packages/minimongo/modify.js

```diff
@@ -349,8 +346,10 @@ var MODIFIERS = {
     if (slice !== undefined) {
       if (slice === 0)
         target[field] = [];  // differs from Array.slice!
-      else
+      else if (slice < 0)
         target[field] = target[field].slice(slice);
+      else 
+        target[field].splice(slice);
```

Comment:
Why not `target[field] = target[field].slice(0, slice)`, to avoid mutating the original array (like above)?